### PR TITLE
Update configuration-of-common-developer-cli-tools-with-cloudflare-wa…

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
Update configuration-of-common-developer-cli-tools-with-cloudflare-warp.md

Included the note on the tools or applications configured with Cloudflare WARP

### CODE REVIEW

File: faqs/configuration-of-common-developer-cli-tools-with-cloudflare-warp.md [hunk 1]
```diff
@@ -1,22 +1,31 @@
 # Configuration of common Developer CLI tools with Cloudflare WARP
 
+This article tells you how to configure the following common applications. To know how to configure any other applications or tools, refer to the [Cloudflare documentation](https://developers.cloudflare.com/cloudflare-one/connections/connect-devices/warp/install-cloudflare-cert).
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply faqs/configuration-of-common-developer-cli-tools-with-cloudflare-warp.md:3]: okay will update accordingly.

File: faqs/configuration-of-common-developer-cli-tools-with-cloudflare-warp.md [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
+
 - [Node.js and NPM](#nodejs-and-npm)
 - [Docker](#docker)
 - [AWS CLI](#aws-cli)
 - [Golang](#golang)
 
+> **Note**:
+>- If you are using the above-mentioned applications or tools, and unable to access SGTS servcies, create a support request. For more information, refer to [create support request](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/raise-an-incident-support-request). 
+>- If you are using other tools and applications, contact [Cloudflare Community Support Forums](https://support.cloudflare.com/hc/en-us).
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply faqs/configuration-of-common-developer-cli-tools-with-cloudflare-warp.md:12]: Sam, "ability to access SGTS service" - this in simple language, isn't it 
"if you are unable to access SGTS service."

[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply faqs/configuration-of-common-developer-cli-tools-with-cloudflare-warp.md]: updated as suggested.

**Review by samloh84 — APPROVED** [review]
Looks good to me.

**Pull request merged**